java/src/jmri/jmrix/loconet/LocoNetConsist.java

```diff
@@ -467,15 +468,15 @@ public void notifyThrottleFound(jmri.DccThrottle t) {
     }
 
     @Override
-    public void notifyFailedThrottleRequest(DccLocoAddress address, String reason) {
+    public void notifyFailedThrottleRequest(LocoAddress address, String reason) {
         notifyConsistListeners(address,
                 ConsistListener.CONSIST_ERROR);
-        removeFromConsistList(address);
+        removeFromConsistList((DccLocoAddress)address);
```

Comment:
Before this cast, you need to make sure the address is a DCCLocoAddress.  You should throw an illegal argument exception if it isn't.